presto-testing/src/main/java/io/prestosql/testing/AbstractTestIntegrationSmokeTest.java

```diff
@@ -243,4 +255,23 @@ private MaterializedResult getExpectedOrdersTableDescription(boolean dateSupport
                     .build();
         }
     }
+
+    @Test
+    public void testInsertForDefaultColumn()
+            throws SQLException
+    {
+        skipTestUnless(supportsDefaultValues());
+        createTableWithDefaultColumns();
+        assertUpdate("INSERT INTO default_table (a) VALUES (1)", 1);
```

Comment:
Use `io.prestosql.testing.sql.TestTable#randomTableSuffix` to make table name unique.